projects/systelab-components/sass/modern/_button.scss

```diff
@@ -22,28 +28,32 @@
   border-radius: 20px;
 }
 
-systelab-toggle-button {
-  .btn { //Toogle buttons
-    &.btn-outline-primary {
-      color: $primary;
-      background-color: transparent;
-      border-color: $primary;
+.btn {
```

Comment:
why did you remove systelab-toggle-button selector?

now it's a generic stile for btn-outline-primary?

Reply:
Yes, these styles should apply also every btn-outline-primary, so I have made them generic